common/src/utils/smtTree.ts

```diff
@@ -47,6 +49,38 @@ export function buildSMT(field :any[], treetype:string): [number, number, SMT]{
     return [count, performance.now() - startTime, tree]
 }
 
+export function exportSMTToJsonFile(count: number, time: number, smt: SMT, outputPath?: string) {
+  const serializedSMT = smt.export();
+  const data = {
+      count: count,
+      time: time,
+      smt: serializedSMT
+  };
+  const jsonString = JSON.stringify(data, null, 2);
+  const defaultPath = path.join(process.cwd(), 'smt.json');
+  const finalPath = outputPath ? path.resolve(process.cwd(), outputPath) : defaultPath;
+
+  fs.writeFileSync(finalPath, jsonString, 'utf8');
+}
+
+export function importSMTFromJsonFile(filePath?: string): SMT | null {
+  try {
+    const jsonString = fs.readFileSync(path.resolve(process.cwd(), filePath), 'utf8');
+    
+    const data = JSON.parse(jsonString);
+    
+    const hash2 = (childNodes: ChildNodes) => (childNodes.length === 2 ? poseidon2(childNodes) : poseidon3(childNodes));
+    const smt = new SMT(hash2, true);
+    smt.import(data.smt);
+    
+    console.log('Successfully imported SMT from JSON file');
+    return smt;
+  } catch (error) {
+      console.error('Failed to import SMT from JSON file:', error);
+      return null;
+  }
+}
```

Comment:
add these functions to export and import smt.json.
this is used to improve test performance